src/torrent/utils/log.cc

```diff
@@ -105,7 +105,8 @@ log_rebuild_cache() {
       return entry.equal_outputs(use_outputs);
     });
     if (cache_itr == log_cache.end()) {
-      cache_itr = log_cache.insert(log_cache.end(), log_cache_entry());
+      log_cache.emplace_back();
+      cache_itr = std::prev(log_cache.end());
```

Comment:
In this case using emplace makes the code less clear.

Reply:
Fixed.